src/main/java/io/seqera/tower/cli/commands/runs/DumpCmd.java

```diff
@@ -193,7 +196,7 @@ private String collectWorkflowLaunch(Long workspaceId) throws ApiException, Json
         String launchId = workflow.getLaunchId();
         Launch launch = (launchId == null) ? null : launchById(workspaceId, launchId);
```

Comment:
I think if there is no launch data we should return null instead of an empty string.

```suggestion
        if (launchId == null) { // nextflow-run workflow, no launch entity available
            return null;
        }
        Launch launch = launchById(workspaceId, launchId);
        return JsonHelper.prettyJson(launch);
```